quirks/websites-with-shared-credential-backends.json

```diff
@@ -217,5 +217,13 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "lufthansa.com",
+        "miles-and-more.com"
+    ],
```

Comment:
And just to be sure, there are login pages on both of these domains that can be used to sign in with the same credentials?

Reply:
Can confirm, use it on my own.